html-forms/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <form action=>
+    <label>Email</label>
```

Comment:
Please include a `for` attribute with a value that matches an `id` attribute on the `input` that correlates to this `label`. Please also do this for each other label in the document.